### PR TITLE
RC on by default and PrivateLink updates to publish on April 8th

### PR DESCRIPTION
**Pull request by bawsome**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by bawsome** [comment]
Hi - we'd like to push these doc updates on April 8th as we're enabling RC on by default for new orgs and also added support for PrivateLink earlier.

**Comment by maycmlee** [comment]
Created [DOCS-7584](https://datadoghq.atlassian.net/browse/DOCS-7584) for docs review. Noted the publish date for April 8th.

**Comment by bawsome** [comment]
Hi @maycmlee we've deployed the changes in production environment today morning. Can you please roll out the doc updates as well?

cc @mellon85

**Comment by maycmlee** [comment]
Thanks! Looping in @ceciliawatt, who is assigned to the docs card, to review and merge.

**Comment by bawsome** [comment]
Hi @ceciliawatt any updates?

File: content/en/agent/remote_config/_index.md [hunk 1]
```diff
@@ -108,13 +108,18 @@ To enable Remote Configuration:
...
 
 3. On the [Remote Configuration][8] page, enable Remote Configuration. This enables Datadog components across your organization to receive configurations from Datadog.
+**Note:** Beginning April 8, 2024, Remote Configuration is on-by-default for:
+* New sub-organizations created by existing Datadog customers who already have enabled RC at the master organization level and where they are creating new sub-organizations in the same Datadog site as their master organization.
+* Organizations created by new Datadog customers.
+  
+To opt-out of Remote Configuration use, see the [opt-out section][23].
 
-4. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key:
+6. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key. If you match the new organization conditions mentioned in step 3, your API keys will be Remote Configuration- capable by default.
```
Comment (cswatt) [inline content/en/agent/remote_config/_index.md:117]:
```suggestion
6. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key. If your new organization fulfills the conditions mentioned in step 3, Remote Configuration is enabled on your API keys be default.
```

How exactly do you enable Remote Configuration on a new or existing API key? Even if it seems obvious, we should explicitly say something like "go to the API keys page and use the toggle"

File: content/en/agent/remote_config/_index.md [hunk 2]
```diff
@@ -215,8 +220,16 @@ The following table describes the meaning of each Tracing library status:
   | NOT DETECTED   | The Tracing library does not support Remote Configuration. To fix this issue, update the Tracing library software to the latest available version. |
   | UNKNOWN   | The Tracing library status is unknown, and it can't be determined if an Agent is associated with the Tracing library. For example, this could be because the Agent is deployed on a fully managed serverless container service like AWS Fargate. |
 
-## Opting out of Remote Configuration at the Agent level
+## Opting out of Remote Configuration 
+To opt-out of Remote Configuration use, you can simply disable Remote Configuration at the organization level. Optionally, you can also disable Remote Configuration capability at the API key level and agent level.
```
Comment (cswatt) [inline content/en/agent/remote_config/_index.md:224]:
```suggestion
To opt-out of Remote Configuration use, you can disable Remote Configuration at the organization level. Optionally, you can also disable Remote Configuration capability at the API key level and Agent level.
```

File: content/en/agent/remote_config/_index.md [hunk 3]
```diff
@@ -215,8 +220,16 @@ The following table describes the meaning of each Tracing library status:
...
 
+### At the Organization level
+Disable Remote Configuration at the organization level on the [Remote Configuration][8] page. This disables Datadog components across your organization to receive configurations from Datadog. You need the [`org_management`][7] permission to disable Remote Configuraiton at the organization level.
+
+### At the API Key level
```
Comment (cswatt) [inline content/en/agent/remote_config/_index.md:229]:
```suggestion
### At the API key level
```

File: content/en/agent/remote_config/_index.md [hunk 4]
```diff
@@ -215,8 +220,16 @@ The following table describes the meaning of each Tracing library status:
   | NOT DETECTED   | The Tracing library does not support Remote Configuration. To fix this issue, update the Tracing library software to the latest available version. |
   | UNKNOWN   | The Tracing library status is unknown, and it can't be determined if an Agent is associated with the Tracing library. For example, this could be because the Agent is deployed on a fully managed serverless container service like AWS Fargate. |
 
-## Opting out of Remote Configuration at the Agent level
+## Opting out of Remote Configuration 
+To opt-out of Remote Configuration use, you can simply disable Remote Configuration at the organization level. Optionally, you can also disable Remote Configuration capability at the API key level and agent level.
 
+### At the Organization level
+Disable Remote Configuration at the organization level on the [Remote Configuration][8] page. This disables Datadog components across your organization to receive configurations from Datadog. You need the [`org_management`][7] permission to disable Remote Configuraiton at the organization level.
```
Comment (cswatt) [inline content/en/agent/remote_config/_index.md:227]:
```suggestion
Disable Remote Configuration at the organization level on the [Remote Configuration][8] page. This disables Datadog components across your organization to receive configurations from Datadog. You need the [`org_management`][7] permission to disable Remote Configuration at the organization level.
```

File: content/en/agent/remote_config/_index.md [hunk 5]
```diff
@@ -215,8 +220,16 @@ The following table describes the meaning of each Tracing library status:
...
+
+### At the API Key level
+Disable the API key of your choice on the [API Keys][24] page. You need the [`api_keys_write`][5] permission to disable Remote Configuration capability on an API key.
```
Comment (cswatt) [inline content/en/agent/remote_config/_index.md:230]:
```suggestion
Disable the API key of your choice on the [API Keys][24] page. You need the [`api_keys_write`][5] permission to disable Remote Configuration on an API key.
```

File: content/en/agent/remote_config/_index.md [hunk 6]
```diff
@@ -215,8 +220,16 @@ The following table describes the meaning of each Tracing library status:
...
+
+### At the Agent level
 Starting with Agent version 7.47.0, `remote_configuration.enabled` is set to `true` by default in the Agent. This setting causes the Agent to request configuration updates from the Datadog site.
 
 To receive configurations from Datadog, you also need to take the following steps:
```
Comment (cswatt) [inline content/en/agent/remote_config/_index.md:235]: Is this section (this line, plus the following three bullet points) in the correct place? seems like it should be moved higher

**Review by cswatt — CHANGES_REQUESTED** [review]

File: content/en/agent/remote_config/_index.md (hunk 6 above)
Comment (bawsome) [reply content/en/agent/remote_config/_index.md:235]: Yes, we believe its the right place as it follows the onboarding process which starts off with org-level, API key level, and Agent-level settings.

File: content/en/agent/remote_config/_index.md (hunk 1 above)
Comment (bawsome) [reply content/en/agent/remote_config/_index.md]: Good Q. The enabling API keys for RC is done through RC onboarding page itself (it's the 2nd step in RC onboarding). Users don't have to go to the API keys page.

**Comment by bawsome** [comment]
Hi @ceciliawatt thanks for the review and feedback. Accepted your suggestions and answered the two questions you had.

**Comment by bawsome** [comment]
Hi @ceciliawatt are you good to push this live today?

**Review by alai97 — APPROVED** [review]

**Pull request merged**